stable/percona-xtradb-cluster/README.md

```diff
@@ -71,6 +71,12 @@ The following table lists the configurable parameters of the Percona chart and t
 | `podAnnotations`           | Pod annotations                    | `{}`                                                       |
 | `resources`                | CPU/Memory resource requests/limits | Memory: `256Mi`, CPU: `100m`                              |
 | `configFiles` | files to write to /etc/mysql/conf.d | see values.yaml |
+| `ssl.enabled`                                | Setup and use SSL for MySQL connections                                                      | `false`                                              |
+| `ssl.secret`                                 | Name of the secret containing the SSL certificates                                           | mysql-ssl-certs                                      |
+| `ssl.certificates[0].name`                   | Name of the secret containing the SSL certificates                                           | `nil`                                                |
```

Comment:
Why we have list here ?

Reply:
This name is duplicate of ssl.secret and I think this chart won't work if those values are different.

Reply:
It's an alternative, not a duplicate.  If you want to use an existing secret, you can use `ssl.secret` without the `ssl.certificates` and you can use `ssl.secret` to specify the name of the secret to save the cert in.

Additionally, I copied the model being used in the stable/mysql chart.

Reply:
But stateful set will use only secret from `.Values.ssl.secret` so certificates name is no needed here.

Reply:
The stateful set uses the secret name, but if the secret doesn't exist, the certificates name is used in `templates/secrets.yaml` as part of the creation there.

See the parity with the `stable/mysql` chart here: https://github.com/helm/charts/tree/master/stable/mysql